runfolder/services.py

```diff
@@ -92,9 +92,12 @@ def _validate_is_being_monitored(self, path):
 
         :raises PathNotMonitored
         """
-        monitored = any([path.startswith(mon) for mon in self._monitored_directories()])
-        if not monitored:
-            raise PathNotMonitored("The path {0} is not being monitored".format(path))
+        monitored = list(self._monitored_directories())
+        is_monitored = any([path.startswith(mon) for mon in monitored])
```

Comment:
You're right that that is how it works now.

I agree that there is no reason for this, and it could be a potential source of confusion. Not sure what the original idea was, I'll just change it.